uiServices/src/homepage.go

```diff
@@ -1,104 +1,105 @@
 package services
```

Comment:
on formatting: run go fmt ./... at the automated-perf-test level and this will help with IDEs having different settings etc.